tests/config/fgcmOutputProductsLatiss.py

```diff
@@ -0,0 +1,21 @@
+import os.path
+
+
+config.connections.cycleNumber = 2
+
+config.doReferenceCalibration = True
+config.referencePixelizationMinStars = 20
+config.referenceMinMatch = 15
+
+from lsst.obs.lsst.filters import LATISS_FILTER_DEFINITIONS
```

Comment:
Can you not use this (rather than set explicitly below)?

Reply:
I only want to define the set of filters that we're using, rather than all possible filters.